services/ingestion/etl/just/csv/src/lib/transform.js

```diff
@@ -65,7 +65,7 @@ const getLocations = record => {
 const getBudget = record => ({
   total_cost: {
     value: Number(record.field_prj_total_budget) || 0,
-    currency: '',
+    currency: 'EUR',
```

Comment:
Normally, there shouldn't be "nothing" here since we extract  `value: Number(record.field_prj_total_budget) || 0,` and all the values in the dataset are in EUR. Do you mean we should check `if value === 0 then currency = ''`?

Reply:
I think it'd be better to store currency only if value is present - it will be consistent with the rest of the updates in this pull request.